common/components/ui/Address.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { toChecksumAddress } from 'ethereumjs-util';
+import NewTabLink from './NewTabLink';
+import { IWallet } from 'libs/wallet';
+import { BlockExplorerConfig } from 'types/network';
+
+interface Props {
+  explorer?: BlockExplorerConfig | null;
+  address?: string;
+  wallet?: IWallet;
+}
```

Comment:
@HenryNguyen5 I couldn't figure out how to make two exclusive sets of props. Doing:
```
interface AddressProps extends BaseProps {
  address: string;
}

interface WalletProps extends BaseProps {
  wallet: IWallet;
}
```
did not work. Any thoughts?